### PR TITLE
rename MsgPack assembly only

### PR DESCRIPTION
**Pull request by anurse**
Part of #1998 . I want to do the assembly first because it's the thing that requires Universe changes and I don't want to just check it in and bail for the day ;P.

It's **much** easier if this cross-repo stuff is done before branching so I really want to get this in today.

This is literally just the cross-repo changes (assembly/package/project names). In-repo changes (namespaces, types, etc.) will come in a PR shortly (today as well, but it's less volatile so I want to front-load this)

Please review ASAP.

TODO:
* [x] Make PR to Universe that updates the assembly name there and takes this commit in a submodule.
* [ ] Merge both PRs "simultaneously"

### CODE REVIEW

File: test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Microsoft.AspNetCore.SignalR.Client.FunctionalTests.csproj [hunk 1]
```diff
@@ -15,7 +15,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Protocols.MsgPack\Microsoft.AspNetCore.SignalR.Protocols.MsgPack.csproj" />
```
Comment (anurse) [inline test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Microsoft.AspNetCore.SignalR.Client.FunctionalTests.csproj]: Already getting it transitively through Test.Utils.

File: test/Microsoft.AspNetCore.SignalR.Common.Tests/Microsoft.AspNetCore.SignalR.Common.Tests.csproj [hunk 2]
```diff
@@ -12,7 +12,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Common\Microsoft.AspNetCore.SignalR.Common.csproj" />
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Protocols.MsgPack\Microsoft.AspNetCore.SignalR.Protocols.MsgPack.csproj" />
```
Comment (anurse) [inline test/Microsoft.AspNetCore.SignalR.Common.Tests/Microsoft.AspNetCore.SignalR.Common.Tests.csproj]: Already getting it transitively through Test.Utils.

File: test/Microsoft.AspNetCore.SignalR.Redis.Tests/Microsoft.AspNetCore.SignalR.Redis.Tests.csproj [hunk 3]
```diff
@@ -11,7 +11,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Protocols.MsgPack\Microsoft.AspNetCore.SignalR.Protocols.MsgPack.csproj" />
```
Comment (anurse) [inline test/Microsoft.AspNetCore.SignalR.Redis.Tests/Microsoft.AspNetCore.SignalR.Redis.Tests.csproj]: Already getting it transitively through Test.Utils.

**Comment by BrennanConroy** [comment]
Update SignalRWorker in benchmarks repo too? Or at least make sure I do it after you merge 😄

**Comment by anurse** [comment]
We'll do that when we finish all the changes.

**Comment by anurse** [comment]
Getting Universe PR ready, once both are approved I'll simul-merge

**Pull request merged**